containerized-data-importer/models/V1NodeSelectorRequirement.ts

```
@@ -35,59 +14,9 @@ export interface V1NodeSelectorRequirement {
-  if (value === undefined) {
-    return undefined;
-  }
-  if (value === null) {
-    return null;
-  }
-  return {
-    key: value.key,
-    operator: value.operator,
-    values: value.values,
-  };
+  operator: "DoesNotExist" | "Exists" | "Gt" | "In" | "Lt" | "NotIn";
```

ESLint

[error] Replace `"DoesNotExist"·|·"Exists"·|·"Gt"·|·"In"·|·"Lt"·|·"NotIn"` with `'DoesNotExist'·|·'Exists'·|·'Gt'·|·'In'·|·'Lt'·|·'NotIn'`

(prettier/prettier)